### ISSUE TITLE
Switch or button in Home Assistant to activate Assist in progress

### ISSUE THREAD

**Issue by celodnb** [rhasspy/wyoming-satellite#220]
I really like the Wyoming satellites, as the accuracy (after some tuning), is close to other commercial offerings from Google and Amazon.

I've also been using some ESP based satellite, and on those I've managed to create a switch in HA, which can activate Assist in progress. This is really handy, as it makes it possible (for example, via an automation) to activate Assist with a smart button or something similar. 

I'd love to have a similar switch for the Wyoming satellites, so I can activate assist without having to use a wake word, so please consider adding.

**Comment by maxi1134** [comment]
This would be great for automation with continued conversation type of things.

**Comment by slyticoon** [comment]
I agree, this feature would complete my transition away from commercial voice assistants. If we could trigger Wyoming sats from automations or node red, we would have actionable notifications, follow up questions, etc... Really important feature. Hope to see it soon.
